src/sage/modular/multiple_zeta_F_algebra.py

```diff
@@ -116,6 +133,8 @@ def basis_f_iterator(n) -> Iterator[tuple]:
 
     - ``n`` -- an integer
 
+    - ``start`` -- optional start index for odd generators (default 3)
```

Comment:
```suggestion
    - ``start`` -- (default: ``3``) start index for odd generators
```
and similar elsewhere.